### PR TITLE
Fix some early imports issues with Django 1.9

### PR DESCRIPTION
**Pull request by yakky**
Fix #4994

### CODE REVIEW

**Comment by yakky** [comment]
@mkoistinen @czpython would you give a second look at this?

**Comment by Chive** [comment]
:+1:

File: cms/appresolver.py [hunk 1]
```diff
@@ -209,6 +211,8 @@ def get_app_patterns():
         current_site = Site.objects.get_current()
     except Site.DoesNotExist:
         current_site = None
+    except OperationalError:
+        return []
```
Comment (czpython) [inline cms/appresolver.py]: isn't this too general? I do think I've gotten this error though..

Comment (yakky) [inline cms/appresolver.py]: It's the only error we get if the database is not ready (e.g.: with no tables in it)

**Comment by czpython** [comment]
:+1:

**Pull request merged**